Enhancing docblock information and adding missing tags

Sjaure opened the pull request:
### 🎫 Ticket

[TICKET_ID]
<!-- Ticket ID, if there's any put it between brackets -->

### 🗒️ Description

<!--
Please describe what you have changed or added
What types of changes does your code introduce?
Bug fix (non-breaking change which fixes an issue)
New feature (non-breaking change which adds functionality)
Include any important information for reviewers
-->

### 🎥 Artifacts <!-- if applicable-->
<!-- 🎥 screencast(s) or 📷 screenshot(s) -->

### ✔️ Checklist
- [ ] Changelog entry in the `readme.txt` file.
- [ ] Code is covered by **NEW** `wpunit` or `integration` tests.
- [ ] Code is covered by **EXISTING** `wpunit` or `integration` tests.
- [ ] Are all the **required** tests passing?
- [ ] Automated code review comments are addressed.
- [ ] Have you added Artifacts?
- [ ] Check the base branch for your PR.
- [ ] Add your PR to the project board for the release.

bordoni commented on src/functions/template-tags/link.php:
```diff
@@ -19,11 +19,13 @@ function tribe_the_prev_event_link( $anchor = false ) {
- * @param bool|string $anchor link text. Use %title% to place the post title in your string.
+ * @since 5.14.0
+ * 
+ * @param bool|string $anchor (optional) The link text. Use %title% to place the post title in your string. Default is false
```
Make sure this is addressed.

bordoni commented:
Add the `Code Review` label and assign yourself.

Sjaure commented:
@bordoni , I added the missing tag, assigned it to myself. Also added the missing punctuation and comments on readme.txt
I guess someone else needs to merge the changes to that file or should I go ahead and resolve it?

Sjaure commented:
@bordoni, I fixed a couple of issues with the changelog file and resolved the conflicts too.

bordoni commented on readme.txt:
```diff
@@ -234,10 +234,17 @@ Remember to always make a backup of your database and files before updating!
 
 = [6.5.1] TBD =
 
+<<<<<<< fix/docblock-tribe_get_prev_event_link
+* Tweak - Add note to `tribe_create_organizer()` docblock to indicate future deprecation.
+* Tweak - Add note to `tribe_create_event()` docblock to indicate future deprecation.
+* Tweak - Add note to `tribe_event_update()` docblock to indicate future deprecation.
+* Tweak - Add information and missing tags to `tribe_get_prev_event_link()`.
+=======
 * Feature - Adapt to using the refactored tribe_asset. Remove some unused asset calls. [TCMN-172]
 * Fix - Remove automatic capitalization for 'View Calendar' text on Calendar List Widget. [TECTRIA-40]
 * Tweak - Add note to `tribe_create_venue()`, `tribe_create_organizer()`, `tribe_create_event()`, `tribe_update_event()`, `tribe_update_venue()`, `tribe_update_organizer()`, and `tribe_delete_venue()` docblocks to indicate future deprecation.
 * Tweak - Add docblocks to `src/Tribe/Featured_Events/Permalinks_Helper.php`.
+>>>>>>> release/T24.asag
```
Resolve the conflict.

bordoni commented:
Move to `release/T24.babayaga`
